code/style.css

```diff
@@ -1,4 +1,253 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background-color: #002233;
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+  padding: 20px;
 }
+
+
+/*** MOBILE MAIN DESIGN ***/
+
+@media (min-width: 1px) and (max-width: 667px) {
+
+  /*** HEADER ***/
+
+  header{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    }
+    
+  nav {
+      display:none;
+    }
+
+  .order-1{
+      order: 1;
+    }
+    
+  .order-2 {
+      order: 2;
+    }
+  
+  .order-3 {
+      order: 3;
+    }
+  
+  .order-4 {
+      order: 4;
+    }
```

Comment:
If you're having them in the order you've written them, you don't need to specify the order